docs/containers-storage.conf.5.md

```diff
@@ -148,31 +126,70 @@ Sets the log level of devicemapper.
     7: LogLevelDebug
 
 **min_free_space**=""
-
-Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
+  Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
 
 **mkfsarg**=""
+  Specifies extra mkfs arguments to be used when creating the base device.
```

Comment:
list of accepted values or pointer to a man page with them?

Reply:
I will pass on this one,  This is an old section, and I think this differs depending on the underlying file system.